### DISCUSSION TITLE
2.3.2 won't make new seed?

### DISCUSSION THREAD

**Discussion by Flunkle** [Trexounay/EnderLilies.Randomizer#7] [Q&A] [answered]
I have tried uninstalling and reinstalling both the randomizer and live split of 1.8.16, 1.8.28 and 1.8.29, but I simply cannot get it to produce the seed. It doesn't matter what settings I put it won't create the EnderLiliesSeed.txt required and when I go back to an old version of the randomizer to get the seed text to be generated it won't edit the text when I go back to the newest one. This has prevented me from using the randomize start feature that I really want to try. If anyone could tell me of anyway to fix this issue, I would greatly appreciate it as I made this account to specifically ask this question.

**Comment by vgfreak95** [comment]
I was going to write earlier this morning, but I'll look into it tomorrow.

1nd Question: What Operating System are you using Windows or GNU Linux?
2nd Question: Have you tried running Livesplit as Administrator? because it may not have the write access to that area for some reason, this should force that access.
3rd Question: Have you tried deleting the seed file yourself and seeing if it would generate again?
4th Question: You are sure you put the DLLs into the proper location? Livesplit/Components should be the folder you put the dlls into.

Just let me know the answer to those questions and what else you may have tried.

**Reply by Flunkle** [reply]
First: Thank you for responding so quickly.
Second: I use Windows.
Third: I haven't tried running it as administrator (which I will try now) but older versions of the randomizer do work which is why I believe it is not LiveSplit.
Fourth: I have tried deleting the seed and would have to go back to older versions to have one appear.
Fifth: I think I should mention that the spoiler tab never showed anything as well I wonder if that's because it has no seed to read?
Sixth: I have checked and made sure the DLLs are in the correct place.

**Reply by Flunkle** [reply]
Running as administrator did not work.

**Reply by Trexounay** [reply]
Hello! What is the old version number you're using that works for generating the .txt file?

**Reply by Flunkle** [reply]
2.2.1

**Reply by vgfreak95** [reply]
I was messing around with the Livesplit version you are talking about, and I want you to try something for me to confirm something.

Go ahead and check the settings you want to use, and then close out of livesplit. Make sure you hit the X Button in the top right to close everything. Once everything is closed, can you check the Components folder and see if there is an EnderLiliesSeed.txt there?

That's whats happening to me so I'm curious if the same will happen to you

**Reply by Flunkle** [reply]
It appears to be working, it does what I want in the settings and spoilers and it loaded in a random area. I'm not sure I did exactly what you said there, but it worked and I can change it around. I am not sure if I even solved any problems or just got lucky but even so thank you so much for your assistance.

**Reply by vgfreak95** [reply, marked as answer]
Yeah there seems to be a bug in the initialization part of the Randomizer Component. This only occurs when the user uses livesplit, and then tries to randomize the settings. For some reason only closing out writes to the file, I found that when I was debugging a bit ago. I'm glad it was fixed and I'm almost certain its a bug. For now enjoy, and everything should work correctly now since its only an initialization of the component bug not writing to the EnderLiliesSeed.txt.

**Reply by Flunkle** [reply]
If I were to hazard a guess, the seed text from the older version might have been making what you suggested about exiting not work because it couldn't override the text because it was from an older script? I tried a couple different seeds and settings and they seemed to work as the pervious builds did just with the new options included. The main difference I noticed was that the spoiler tab would update as I changed the setting when before it didn't change at all. I hope this information helps with whatever the issue was.

**Reply by Flunkle** [reply]
Ah you replied to that other message while I was typing and I didn't notice sorry.
